test/runtests.jl

```diff
@@ -30,7 +30,11 @@ logfile = joinpath(tempdir(), "anon.log")
 end
 data = SnoopCompile.read(logfile)
 pc = SnoopCompile.parcel(reverse!(data[2]))
-@test length(pc[:Base]) <= 1
+if Base.VERSION < v"1.10"
+    @test length(pc[:Base]) <= 1
+else
+    @test !haskey(pc, :Base)
+end
```

Comment:
Could you explain why this test case stopped working with v1.10 and later? Otherwise could you mark it as `broken` or simply leave it unchanged?

Reply:
I provide some more detail on this in #380. This test was introduced by #27 to fix #26. In particular, #27 avoided making precompile statements for lines that contain anonymous functions. 

On Julia 1.6.7, three statements are generator from the `@snoopc` usage. Two of them contain anonymous functions and are correctly ignored.
```
4020491	"Tuple{typeof(Base.map), Function, Array{Int64, 1}}"
4020188	"Tuple{Type{Base.Generator{I, F} where F where I}, Main.var"#1#2", Array{Int64, 1}}"
24086328	"Tuple{typeof(Base.collect_similar), Array{Int64, 1}, Base.Generator{Array{Int64, 1}, Main.var"#1#2"}}"
```

On Julia 1.10.2, only two statements are generated and they are both correctly ignored.
```
3918146	"Tuple{Type{Base.Generator{I, F} where F where I}, Main.var"#1#2", Array{Int64, 1}}"
20688430	"Tuple{typeof(Base.collect_similar), Array{Int64, 1}, Base.Generator{Array{Int64, 1}, Main.var"#1#2"}}"
```

The only difference is that `Tuple{typeof(Base.map), Function, Array{Int64, 1}}` is already precompiled in Julia 1.10.2. This has nothing to do with the purpose of the test.

Reply:
Alright, thanks so much for your explanation. All that sounds reasonable.